cli/config.yml

```diff
@@ -5,21 +5,7 @@ disableKubernetes: false
 disableIG: false
 disableCustomTargetSelection: false
 packages:
-  - id: core
-    name: Core package
-  - id: client
-    name: OpenCR Client Registry
-  - id: elastic-analytics
-    name: Data Analytics package using the elastic stack
-  - id: elastic-pipeline
-    name: Data pipeline package using the elastic stack
-  - id: emr
-    name: Electronic Medical Record Package
-  - id: facility
-    name: Facility Registry
-  - id: hmis
-    name: health-management-information-system
-  - id: healthworker
-    name: healthworker
-  - id: mcsd
-    name: mCSD use case package
+  - id: example
```

Comment:
^^Above, you should change the image to jembi/go-cli-test-image. For whatever reason, running godog does a pre-test check on all go files, and removing the config file and keeping it in the 'test-conf' results in the test breaking (it doesn't even start). So, I recommend removing test conf altogether and making the relevant changes to build-cli.sh since there will no longer be test-conf file.

Reply:
👍